src/components/MovieFullDescription/MovieFullDescription.jsx

```diff
@@ -1,46 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { TextWrap, Text } from '../MovieList/MovieListItem/style';
+import {
+  Label, LabelGroup, TitleLabel, Title,
+} from '../MovieList/MovieListItem/style';
+
+function checkReleaseDate(releaseDate) {
```

Comment:
Please, change the function name.
I mean, if the function is called like checkSmth() it should validate some data and return `true` of `false` depends on the result.
But your function returns release date (some string) depends on some statements. It better name like getReleaseDate() or adaptReleaseDate() or fixReleaseDate()